transcom/mymove
# [MB-6195] Update users edit page in the admin app

## Description

Added the active field to the UI, and some non-editable fields. Also added cypress test to check for new fields. 

## Acceptance test 

Acceptance must be done on staging. 

If you want to do these tests during review, you need to do it locally. Which you can, you just need to use a real login.gov account for the `RevokEE` not local sign in. 

LOGIN AS REVOKER
  1. Login as admin on staging, let's call that the `RevokER` account for these instructions. 
     You are going to use this account to deactivate and reactivate the test account from the apps.
     When you deactivate a user, you want to see 
- that they get automatically logged out in their window (either on refresh or they will start getting failures if they continue to use the UI) 
- AND that they can't log back in. 

TEST MILMOVE USER DEACTIVATION
  2. Create a milmove user in separate browser on my.stg.move.mil - let's call that user the `RevokEE`
  3. Find that milmove user using the `Revoker's` window and deactivate the account. 
  4. Check that `Revokee`, on refresh, gets logged out automatically
  5. Check that `Revokee`, cannot log back in
  6. Find that `Revokee` using the `Revoker's` admin window and reactivate the account. 
  7. Check that `Revokee` user CAN log back in. 

TEST OFFICE USER DEACTIVATION
8. - Use `Revoker` account to give that `Revokee` office and admin accounts. Use the create button on the `Office User` and `Admin User` page, and the email of the `Revokee`. 
9. - Login to office.stg.move.mil  and do 3-7 above to check that you can deactivate the office access of `Revokee` and kick them out. 

TEST ADMIN USER DEACTIVATION
10. - Login to admin.stg.move.mil  and do 3-7 above to check that you can deactivate the admin access of `Revokee` and kick them out. 

Note - you can sort of do these all together with one deactivate, but it's hard to keep track of when there are idle timers that will also kill the sessions.


- 
## Code Review Verification Steps


## References

* [Jira story](https://dp3.atlassian.net/browse/mb-6195) for this change

## Screenshots

New fields
![image](https://user-images.githubusercontent.com/40670806/106815160-507c9900-6628-11eb-8b88-391b69f18d78.png)
